### PR TITLE
Replace cpx dependency with cpx2

### PR DESCRIPTION
**Pull request by skirby1996**
The cpx package is no longer being mantained, and was forked into the [cpx2](https://github.com/bcomnes/cpx2) package. This PR replaces cpx with cpx2 so that we can stop suppressing npm audit failures from the cpx package.

### CODE REVIEW

**Review by aruniverse — COMMENTED** [review]
Any reason we don't bump to the latest of cpx2  (4.0?) 
https://www.npmjs.com/package/cpx2/v/4.0.0
https://github.com/bcomnes/cpx2/blob/master/CHANGELOG.md

**Review by pmconne — APPROVED** [review]

**Comment by skirby1996** [comment]
> Any reason we don't bump to the latest of cpx2 (4.0?) https://www.npmjs.com/package/cpx2/v/4.0.0 https://github.com/bcomnes/cpx2/blob/master/CHANGELOG.md

The latest version of cpx2 requires node 14 or greater, 3.0.2 is the latest version of cpx2 that still supports node 12.

**Comment by aruniverse** [comment]
> > Any reason we don't bump to the latest of cpx2 (4.0?) [npmjs.com/package/cpx2/v/4.0.0](https://www.npmjs.com/package/cpx2/v/4.0.0) [bcomnes/cpx2@`master`/CHANGELOG.md](https://github.com/bcomnes/cpx2/blob/master/CHANGELOG.md?rgh-link-date=2021-11-16T16%3A48%3A58Z)
> 
> The latest version of cpx2 requires node 14 or greater, 3.0.2 is the latest version of cpx2 that still supports node 12.

My bad, for some reason I thought we dropped support for node 12, it is approaching EOL

**Review by aruniverse — APPROVED** [review]

**Review by MichaelBelousov — APPROVED** [review]

**Review by bsteinbk — APPROVED** [review]

**Review by calebmshafer — APPROVED** [review]

**Review by grigasp — APPROVED** [review]

**Comment by calebmshafer** [comment]
@skirby1996 mind updating the audit.js file to no longer ignore this cve?

**Review by mdastous-bentley — APPROVED** [review]

File: core/backend/package.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
     "@types/ws": "^6.0.4",
     "chai": "^4.1.2",
     "chai-as-promised": "^7",
-    "cpx": "^1.5.0",
+    "cpx2": "^3.0.0",
```
Comment (ColinKerr) [inline core/backend/package.json:70]: Why are we updating to 3.x if 4.0 is the latest?

Comment (calebmshafer) [reply core/backend/package.json:70]: See this comment, https://github.com/iTwin/itwinjs-core/pull/2714#issuecomment-970462569.

It's due to use still supporting Node 12.

**Review by ColinKerr — APPROVED** [review]

**Pull request merged**